src/components/OnboardingMenu/components/OnboardingMenuItem/frequentFunctions/OnboardingMenuItemText/OnboardingMenuItemText.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Do we really need to extract `MenuItemText` and `MenuItemImage` to separate components?I don't think so, let's keep them in the `MenuItem` component

Reply:
1) What is the problem?
2) Different components look more convenient for `OnboardingMenuItem`